daemons/controld/controld_messages.c

```diff
@@ -945,7 +945,8 @@ handle_shutdown_ack(xmlNode *stored_msg)
         return I_NULL;
     }
 
-    if ((fsa_our_dc == NULL) || (strcmp(host_from, fsa_our_dc) == 0)) {
+    if ((controld_globals.dc_name == NULL)
+        || (strcasecmp(host_from, controld_globals.dc_name) == 0)) {
```

Comment:
Not for this PR, but this would simplify using null_matches

Reply:
oh, yeah